IdentityCore/src/oauth2/token/MSIDAccessToken.m

```diff
@@ -109,6 +109,12 @@ - (instancetype)initWithTokenCacheItem:(MSIDCredentialCacheItem *)tokenCacheItem
     {
         _expiresOn = tokenCacheItem.expiresOn;
         _extendedExpiresOn = tokenCacheItem.extendedExpiresOn;
+        NSMutableDictionary *serverInfo = [NSMutableDictionary dictionaryWithDictionary:_additionalServerInfo];
+        [serverInfo removeObjectForKey:MSID_EXTENDED_EXPIRES_ON_CACHE_KEY];
```

Comment:
why is this necessary?

Reply:
The base class sets extendedExpiresOn in the dictionary, while MSIDAccessToken has a specific property for this. Having the value in both places is ambiguous.